### ISSUE TITLE
Some routes don't render unless another route is selected

### ISSUE THREAD

**Issue by jimleeisme** [trynmaps/tryn-react#62]
The routes also don't show any stops or buses.
Problematic routes:

- BUS
- E
- K/T
- K-OWL
- L-OWL
- M-OWL
- N-OWL
- T-OWL
- 7R
- 76X
- 81X
- 82X
- 88
- 90
- 91

**Comment by EddyIonescu** [comment]
Thanks for adding this - seems like a good starter ticket :)

**Comment by EddyIonescu** [comment]
@jimleeisme @fulin426 this actually seems to only be a bug for the K/T and E lines - they might be something different in the API response (the bug is where the web-app is displaying the vehicles from the API response).
- BUS is for when muni-metro is down, so not very often, exists but is hard to test for
- 7R doesn't exist anymore - we should use a newer GTFS
- 76X only runs like 4 times on weekends - it works
- 81X runs only on weekday mornings - it works
- 82x runs only on weekday peak - it works
- 83x runs only on weekday peak - it works
- 88 doesn't exist anymore
- 90 and 91 are middle of the night routes - they both work
- E works but seems to be pretty infrequent (on some days it just doesn't run at all - guess it's due to the historic streetcar+cable-car shortage)

In summary: 
- dashes and foreslashes are likely messing things up (on the web-app side, as they work in the API)

**Comment by EddyIonescu** [comment]
As for stops not showing, that's an unrelated bug (occurs when there are no vehicles running on that route), it's being tracked here (and is also a good first issue :) https://github.com/trynmaps/opentransit-map/issues/74
